components/RegistrationForm/__tests__/RegistrationForm.test.js

```diff
@@ -171,26 +169,22 @@ describe('RegistrationForm', () => {
       expect(OperationCodeAPIMock.history.post.length).not.toBeGreaterThan(0);
     });
 
-    // + 1 because of always-present form Alert (conditionally rendered text)
-    expect(wrapper.find('Alert').children()).toHaveLength(Object.keys(initialValues).length + 1);
+    // All fields + 1 because of always-rendered form error (as opposed to all the field errors)
+    expect(wrapper.find('Alert').children()).toHaveLength(
+      Object.keys(invalidFormValues).length + 1,
+    );
   });
 
   it('should show "email already registered" message for dupe email registration', async () => {
-    const testUser = {
-      email: 'kylemh.email12@gmail.com',
-      password: 'Testing123!',
-      firstName: 'Test',
-      lastName: 'User',
-      zipcode: 90630,
-    };
+    const existingUser = await mockUser('kylemh.email12@gmail.com');
 
     OperationCodeAPIMock.onPost('users', {
       user: {
-        email: testUser.email,
-        password: testUser.password,
-        first_name: testUser.firstName,
-        last_name: testUser.lastName,
-        zip: testUser.zipcode,
+        email: existingUser.email,
```

Comment:
This is where something like `hump`, a util, or decamelizing in backend would be helpful and make things cleaner for client. I could always create that util for decamelizing too since none of these values are nested and it would seem easy.

Reply:
You mean on the back-end? You're free to, but it's going alright for now!